addon/index.js

```diff
@@ -0,0 +1,168 @@
+import { guidFor } from '@ember/object/internals';
+import { assert } from '@ember/debug';
+
+/**
+  This class is used internally by Ember and Ember Data.
+  Please do not use it at this time. We plan to clean it up
```

Comment:
Seems wrong?  Don't we actually want folks to feel free to use this?

Reply:
yes, I deliberately just imported the current state for now and will continue iterating on top of it